src/main/java/controller/GameController.java

```diff
@@ -0,0 +1,85 @@
+package controller;
+
+import java.util.Scanner;
+
+import model.Car;
+import model.Racing;
+
+public class GameController {
+    private static final String START_GAME_MESSAGE = "자동차 경주 게임을 시작합니다.";
+    private static final String PLEASE_INPUT = "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)";
+    private static final String HOW_MANY_TIMES_TRY_MESSAGE = "시도할 회수는 몇 회인가요?";
+    private static final String IS_WIN = " 이(가) 최종 우승하였습니다.";
+    public static final String PLEASE_INPUT_AGAIN = "잘못된 입력입니다. 다시 입력해주세요.";
```

Comment:
```suggestion
    private static final String PLEASE_INPUT_AGAIN = "잘못된 입력입니다. 다시 입력해주세요.";
```

``PLEASE_INPUT_AGAIN`` 상수는 GameController 내에서만 사용됩니다. 접근 제어자를 **private**로 변경해주는게 좋을 것 같습니다.

Reply:
수정하는 과정에서 놓친 부분인 것 같습니다. 감사합니다.